server/rpm/pbench-server.spec.j2

```diff
@@ -112,39 +113,42 @@ chown -R pbench.pbench /%{installdir}
 
 %preun
 
+# FIXME:  what about these?
+#   - the results of the `pip3 install --user`
+#   - the pbench user account
```

Comment:
Interesting question. Historically, the `.spec` file *depended* on the `pbench` user account existing, but it was actually created by an Ansible role. However, because we did depend on it, I added `.spec` code to create it to be sure it was there; and maybe that means we should also *delete* it on uninstall? And, if so, that would take with it all of the `--user` Python modules.

We could extend this cleanup discussion to the archive/result directory trees; arguably the archive tree implicitly belongs to the user, and contains potentially valuable and irreplacable user data (the raw tarballs), but the unpacked results and links are arguably ours.

I think for right now I'd be inclined to say that we should drop the pbench user/group but leave the archive/result trees alone ... except they're *owned* by pbench, which means we'd probably need to `chown` them all first to ... `root`??

Maybe: can we determine any standard Fedora practice for installers that create accounts or groups? Is it written down in the packaging rules?

Reply:
Tracked in #2481.